### PR TITLE
Testing/2.0.8

### PR DESCRIPTION
**Pull request by Croydon**

### CODE REVIEW

**Review by uilianries — APPROVED** [review]
LGTM

**Comment by Croydon** [comment]
Urgh. @uilianries Any ideas where this is coming from? https://travis-ci.org/bincrafters/conan-sdl2/jobs/449155334#L1758

**Comment by uilianries** [comment]
looks like is related to python2.7-minimal package. conanio Docker images are running /usr/bin/python as python 3.7. Maybe will be necessary install python2.7 full.

**Comment by Croydon** [comment]
It only fails in the GCC 4.9 containers (with Python 3.6) 🤔

**Comment by uilianries** [comment]
Both gcc-4.8 and gcc-4.9 use old Ubuntu distros, I used some workarounds to install python3.6. going to check now. thanks for reporting

File: conanfile.py [hunk 1]
```diff
@@ -197,7 +203,14 @@ def source(self):
         tools.get(source_url)
         extracted_dir = "SDL2-" + self.version
         os.rename(extracted_dir, self._source_subfolder)
+<<<<<<< HEAD
```
Comment (SSE4) [inline conanfile.py]: conflict marker

File: conanfile.py [hunk 2]
```diff
@@ -197,7 +203,14 @@ def source(self):
...
+        tools.replace_in_file(
+                os.path.join(self._source_subfolder, 'CMakeLists.txt'),
+                'install(FILES ${SDL2_BINARY_DIR}/libSDL2.${SOEXT} DESTINATION "lib${LIB_SUFFIX}")',
+                '')
+=======
```
Comment (SSE4) [inline conanfile.py]: conflict marker

File: conanfile.py [hunk 3]
```diff
@@ -197,7 +203,14 @@ def source(self):
...
         tools.patch(base_path=self._source_subfolder, patch_file="cmake.patch")
+>>>>>>> stable/2.0.8
```
Comment (SSE4) [inline conanfile.py]: conflict marker

File: conanfile.py [hunk 4]
```diff
@@ -265,16 +278,25 @@ def configure_cmake(self):
         elif self.settings.os == "Windows":
             cmake.definitions["DIRECTX"] = self.options.directx
 
+<<<<<<< HEAD
```
Comment (SSE4) [inline conanfile.py]: conflict marker

File: conanfile.py [hunk 5]
```diff
@@ -265,16 +278,25 @@ def configure_cmake(self):
...
+        cmake.configure(build_dir=self._build_subfolder)
+=======
```
Comment (SSE4) [inline conanfile.py]: conflict marker

File: conanfile.py [hunk 6]
```diff
@@ -265,16 +278,25 @@ def configure_cmake(self):
...
+>>>>>>> stable/2.0.8
         return cmake
 
     def build_cmake(self):
-        cmake = self.configure_cmake()
+        cmake = self._configure_cmake()
         cmake.build()
 
     def package(self):
+<<<<<<< HEAD
```
Comment (SSE4) [inline conanfile.py]: conflict marker

File: conanfile.py [hunk 7]
```diff
@@ -265,16 +278,25 @@ def configure_cmake(self):
...
+        cmake = self._configure_cmake()
+        cmake.install()
+=======
         cmake = self.configure_cmake()
         cmake.install(build_dir=os.path.join(self.build_folder, self._build_subfolder))
+>>>>>>> stable/2.0.8
```
Comment (SSE4) [inline conanfile.py]: conflict marker

**Review by SSE4 — CHANGES_REQUESTED** [review]
this one seems to be messed up, there are conflict markers within files, please fix it

**Comment by SSE4** [comment]
@Croydon there are conflict markers within conanfile, please fix it

**Comment by Croydon** [comment]
@SSE4 Please read the pull request history. I didn't even push this and don't know what this is about

**Comment by SSE4** [comment]
@Croydon so what should we do? revert last commit?

**Comment by Croydon** [comment]
@solvingj ^

**Comment by solvingj** [comment]
Looks like was a mistake. Apologies, need to look more closely.

**Pull request merged**